packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -107,6 +109,12 @@ class AnalyzeCommand extends FlutterCommand {
       hide: !verboseHelp,
     );
 
+    argParser.addFlag('ios',
```

Comment:
would we ever want to grab the data from both ios and android subprojects at the same time?

Reply:
The dashboard would grab both data to display a comprehensive view.

but I imagine there will be case we only want the date for a certain platform.